vivo-project/Vitro
# VIVO-3871 write TBox data in Add/Remove RDF with RDFService so as not to "shred" statements with blank nodes

**[VIVO-3871](https://github.com/vivo-project/VIVO/issues/3871)**:

# What does this pull request do?
Adds/removes TBox data on Add/Remove RDF page using the same technique already used for ABox data.  Because blank nodes are written to RDFService as part of a single ChangeSet, blank node statements remain intact.

# How should this be tested?
Upload dataset.ttl example file from linked issue.  Observe that triples are not shredded.  An easy way to do so is to issue this query from the SPARQL query page.  Leave syntax set to RDF/XML and you will see an intact nested superclass definition.
```
DESCRIBE ?x WHERE {
  ?x a owl:Class
  FILTER(REGEX(STR(?x), "vivo.mydomain.edu"))
}
```

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
